Documentation/README_MRTKStandardShader.md

```diff
@@ -83,15 +83,15 @@ For static lighting, the shader will respect lightmaps built by Unity's [Lightma
 
 * See [Proximity Light](Rendering/ProximityLight.md)
 
-## Lightweight Scriptable Render Pipeline support
+## Universal Render Pipeline support
 
-The MRTK contains an upgrade path to allow developers to utilize Unity's Lightweight Scriptable Render Pipeline (LWRP) with MRTK shaders. Tested in Unity 2019.1.1f1 and Lightweight RP 5.7.2 package. or instructions on getting started with the LWRP, see [this page](https://docs.unity3d.com/Packages/com.unity.render-pipelines.lightweight@5.10/manual/getting-started-with-lwrp.html).
+The MRTK contains an upgrade path to allow developers to utilize Unity's Universal Render Pipeline (URP) with MRTK shaders. Tested in Unity 2019.4.13f1 and Universal RP 7.3.1 package. For instructions on getting started with the URP, see [this page](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@7.3/manual/index.html).
 
-To perform the MRTK upgrade, select: **Mixed Reality Toolkit -> Utilities -> Upgrade MRTK Standard Shader for Lightweight Render Pipeline**
+To perform the MRTK upgrade, select: **Mixed Reality Toolkit -> Utilities -> Upgrade MRTK Standard Shader for Universal Render Pipeline**
 
-![lwrp upgrade](../Documentation/Images/MRTKStandardShader/MRTK_LWRPUpgrade.jpg)
+![urp upgrade](../Documentation/Images/MRTKStandardShader/MRTK_URPUpgrade.jpg)
```

Comment:
It looks like the text in this image still says `Upgrade MRTK Standard Shader for Lightweight Render Pipeline`, could you update it?

Reply:
Right, missed this. Thanks, will fix it.